packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Changed
+- Update `filterState` to remove navigations, template, and a subset of
+  autocomplete data when `history.length === 0`.
```

Comment:
Should specify the configuration, ie `when history configuration is set to length: 0` or something

Reply:
Updated. Resolving.